tools/ALARAJOYWrapper/njoy_tools.py

```diff
@@ -295,20 +302,27 @@ def run_njoy(element, A, matb):
                         text=True, capture_output=True)
 
     # If the run is successful, log out the output
-    # and make a copy of the file as a .GENDF file
+    # and make a copy of the file as a GENDF or PENDF file
     if not result.stderr:
-        for fileinfo in file_metadata.values():
-            save_path = dir / fileinfo['dir'] / f'tendl_2017_{element}{str(A).zfill(3)}'
-
-            # Ensure existence of save directory for PENDF/GENDF files
-            (dir / fileinfo['dir']).mkdir(exist_ok=True)
+        fileinfo = file_metadata[file_capture]
+        save_path = (
+            dir / 
+            fileinfo['dir'] / 
+            f'tendl_2017_{element}{str(A).zfill(3)}'
+        )
 
-            fileinfo['save'] = save_path.with_suffix(fileinfo['ext'])
-            Path(f'tape{fileinfo['tape']}').rename(fileinfo['save'])
-            if fileinfo['ext'] == '.gendf':
-                ensure_gendf_markers(fileinfo['save'], matb)
+        # Ensure existence of save directory for PENDF/GENDF files
+        (dir / fileinfo['dir']).mkdir(exist_ok=True)
 
-    return file_metadata['GENDF']['save'], result.stderr
+        fileinfo['save'] = save_path.with_suffix(fileinfo['ext'])
+        tape_save = Path(f'tape{fileinfo['tape']}')
+        if fileinfo['ext'] == '.gendf':
+            tape_save.rename(fileinfo['save'])
+            ensure_gendf_markers(fileinfo['save'], matb)
+        else:
+            shutil.copy(tape_save, fileinfo['save'])
```

Comment:
I'm pretty sure we can use `Path` for this

Reply:
Seems like something we should be able to figure out.

It looks like when there does happen to be an error, this method doesn't actually return anything.  ie. there is no `return` line when an error occurs and therefore, we don't get much insight about what the error is.